ProcSDF/GUI/Inspector.cpp

```diff
@@ -94,6 +96,24 @@ void Inspector::drawRenderingSettings()
 
 void Inspector::drawNodeGraphSettings()
 {
+	if (ImGui::Button("Add custom node"))
+	{
+		std::pair<bool, std::string> l_pickFileResult = OS::pickFile();
+		if (l_pickFileResult.first)
+		{
+			std::string l_filePath = l_pickFileResult.second;
+			CustomNode* l_customNode = new CustomNode(l_filePath);
+			if (l_customNode->isMalformed())
```

Comment:
moreover, we are not checking if the file submitted has a compilation error. which leads to stalling of the program if there is one. We need to fix this in the next pr.

Reply:
#34 opened